src/theme.js

```diff
@@ -0,0 +1,103 @@
+import { createMuiTheme } from "@material-ui/core/styles"
+
+// A custom theme for this app
+const theme = createMuiTheme({
+  palette: {
+    primary: {
+      main: "#292F36",
+    },
+    secondary: {
+      main: "#F7FFF7",
+    },
+    tertiary: {
+      main: "#47CCCC",
+    },
+    background: {
+      default: "#F7FFF7",
+      navbar: "#FFFFFF",
+      footer: "#292F36",
+    },
+  },
+  typography: {
+    fontFamily: ["Proxima Nova"],
+    fontSize: 16,
+    h1: {
+      fontFamily: "Garamond",
+      fontSize: "1.75rem",
+      lineHeight: "2.25rem",
+      letterSpacing: "0.05em",
+    },
+    h2: {
+      fontFamily: "Garamond",
+      fontSize: "1.375rem",
+      lineHeight: "1.625rem",
+      letterSpacing: "0.05em",
+    },
+    h3: {
+      fontFamily: "Garamond",
+      fontSize: "1rem",
+      lineHeight: "1.25rem",
+      letterSpacing: "0.05em",
+    },
+    subtitle1: {
+      fomtFamily: "Proxima Nova",
```

Comment:
The property `fontFamily` is misspelled.

Reply:
Great catch @JZenk and thanks for jumping in!